airflow/models.py

```diff
@@ -2567,6 +2570,7 @@ def __init__(
         self._concurrency = concurrency
         self._pickle_id = None
 
+        self._description = description
```

Comment:
I don't think there's a need to make description private here though I wouldn't want to get in a philosophical discussion about that :)